### PR TITLE
Pull Request for merging from Feature to Development

### PR DESCRIPTION
**Pull request by alexawesomecode**
Hello

We worked on all the project requirements:

- We created session controller and implemented logic for signing users and remembering them
- Created posts database
- We created posts controller and views for showing lists of posts and authors name
- Users can sign in and logout and also create posts

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Rails.application.routes.draw do
+  # get 'sessions/new'
+  get  'sessions/new',   to: 'sessions#new'
+  post 'sessions/new',   to: 'sessions#create'
+  root 'sessions#index'
+  post 'posts/new',      to: 'posts#create'
```
Comment (Ispirett) [inline config/routes.rb]: You don't need this its duplicate code

File: config/routes.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  resources :sessions
```
Comment (Ispirett) [inline config/routes.rb:7]: [optional] you should specify only `new and create`

**Review by Ispirett — CHANGES_REQUESTED** [review]
## Status Required :+1:  :100: 
Greate work @alexawesomecode @phalado this is great.
There are just some things missing.

- [x]  There should be a user controller with a new and create action, see [requirement](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/authentication/sign_up_requirement.png) and  [requirement](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/authentication/sign_up_requirement.png)
- [x] It has a sign up form created with #form_for in `app/views/users` [required](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/authentication/sign_up_requirement.png)

File: app/controllers/user_controller.rb [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+                     password: params[:user][:password],
+                     password_confirmation: params[:user][:password_confirmation])
+    if @user.save
+      flash.now[:info] = "Account created."
+      redirect_to root_url
+    else
+      render 'new'
+    end
+  end
+end
```
Comment (samgaco) [inline app/controllers/user_controller.rb:18]: :+1:

File: app/views/user/new.html.erb [hunk 4]
```diff
@@ -0,0 +1,20 @@
+<h1> Sign up</h1>
+<div class="row">
+  <div class="col-md-6 col-md-offset-3">
+    <%= form_for(:user) do |f| %>
+      <%= f.label :name %>
+      <%= f.text_field :name, class: 'form-control' %>
+      <br>
```
Comment (samgaco) [inline app/views/user/new.html.erb:7]: :medal_military:

**Review by samgaco — APPROVED** [review]
Everything looks ok now. You can move on :+1: 

---------------------------------

* ###### STATUS OF THE PROJECT: APPROVED :heavy_check_mark:

**Pull request merged**